packages/form-config/src/components/DraftFiles/index.tsx

```diff
@@ -0,0 +1,325 @@
+import { Button, Row, Col } from 'reactstrap';
+import { ManifestFilesTypes } from '../../ducks/manifestFiles';
+import { Redirect } from 'react-router';
+import {
+  MAKE_RELEASE_LABEL,
+  FILE_NAME_LABEL,
+  FILE_VERSION_LABEL,
+  IDENTIFIER_LABEL,
+  MODULE_LABEL,
+  DOWNLOAD_LABEL,
+  FIND_DRAFT_RELEASES_LABEL,
+  CREATED_AT_LABEL,
+  UPOL0AD_FILE_LABEL,
+} from '../../constants';
+import { Cell } from 'react-table';
+import { formatDate, downloadManifestFile } from '../../helpers/utils';
+import { Link } from 'react-router-dom';
+import { Dictionary } from '@onaio/utils';
+
+/** Register reducer */
+reducerRegistry.register(draftReducerName, DraftFilesReducer);
+
+/** default props interface */
+export interface DraftsDefaultProps extends SearchBarDefaultProps {
+  clearDraftFiles: typeof removeManifestDraftFiles;
+  createdAt: string;
+  data: ManifestFilesTypes[];
+  downloadLabel: string;
+  drillDownProps: DrillDownProps;
+  fetchDraftFiles: typeof fetchManifestDraftFiles;
+  fileNameLabel: string;
+  fileVersionLabel: string;
+  identifierLabel: string;
+  makeReleaseLabel: string;
+  moduleLabel: string;
+  uploadFileLabel: string;
+}
+
+/** manifest Draft files props interface */
+export interface ManifestDraftFilesProps extends DraftsDefaultProps, FormConfigProps {
+  downloadEndPoint: string;
+  formUploadUrl: string;
+  manifestEndPoint: string;
+  releasesUrl: string;
+  uploadTypeUrl: string;
+}
+
+/**
+ * Component ManifestDraftFiles
+ *
+ * @param {Dictionary} props - component props
+ * @returns {Element} - rendered element
+ */
+const ManifestDraftFiles = (props: ManifestDraftFilesProps): JSX.Element => {
+  const {
+    baseURL,
+    endpoint,
+    getPayload,
+    LoadingComponent,
+    data,
+    debounceTime,
+    placeholder,
+    fetchDraftFiles,
+    clearDraftFiles,
+    customAlert,
+    downloadEndPoint,
+    releasesUrl,
+    manifestEndPoint,
+    makeReleaseLabel,
+    identifierLabel,
+    fileNameLabel,
+    fileVersionLabel,
+    moduleLabel,
+    downloadLabel,
+    createdAt,
+    uploadFileLabel,
+    formUploadUrl,
+    uploadTypeUrl,
+    drillDownProps,
+  } = props;
+
+  const [loading, setLoading] = useState(false);
+  const [stateData, setStateData] = useState<ManifestFilesTypes[]>(data);
+  const [ifDoneHere, setIfDoneHere] = useState(false);
+
+  useEffect(() => {
+    /** get manifest Draftfiles */
+    setLoading(true);
+    /* eslint-disable-next-line @typescript-eslint/camelcase */
+    const params = { is_draft: true };
+    const clientService = new OpenSRPService(baseURL, endpoint, getPayload);
+    clientService
+      .list(params)
+      .then((res: ManifestFilesTypes[]) => {
+        fetchDraftFiles(res);
+      })
+      .catch((error) => {
+        if (customAlert) {
+          customAlert(String(error), { type: 'error' });
+        }
+      })
+      .finally(() => setLoading(false));
+  }, [baseURL, endpoint, getPayload, customAlert, fetchDraftFiles]);
```

Comment:
Can we re-evaluate this. Am not sure if we would want to make a call to API if `customAlert` changes.

Reply:
@ciremusyoka eslint requires that the customAlert is included in the dependency array